### PR TITLE
lang: update German translation

### PR DESCRIPTION
**Pull request by Waboodoo**
Updated the German translation to include the missing strings.

### CODE REVIEW

**Comment by saghul** [comment]
@csett86 mind taking a look to check it's consistent with existing translations?

**Comment by csett86** [comment]
Thanks for the ping! I'll adjust one thing to have it consistent with the overall German translation: Teilnehmer -> Person

I'll check if I can do it myself, otherwise I'll add line comments

File: lang/main-de.json [hunk 1]
```diff
@@ -305,6 +306,8 @@
         "contactSupport": "Support kontaktieren",
         "copied": "Kopiert",
         "copy": "Kopieren",
+        "demoteParticipantDialog": "Sind Sie sicher, dass Sie diesen Teilnehmer zu den Gästen verschieben möchten?",
```
Comment (csett86) [inline lang/main-de.json:309]:
```suggestion
        "demoteParticipantDialog": "Sind Sie sicher, dass Sie diese Person zu den Gästen verschieben möchten?",
```

File: lang/main-de.json [hunk 2]
```diff
@@ -937,6 +946,7 @@
         "or": "oder",
         "premeeting": "Vorschau",
         "proceedAnyway": "Trotzdem fortsetzen",
+        "recordingWarning": "Dieser Anruf wird möglicherweise von anderen Teilnehmern aufgezeichnet",
```
Comment (csett86) [inline lang/main-de.json:949]:
```suggestion
        "recordingWarning": "Diese Konferenz wird möglicherweise von anderen Personen aufgezeichnet",
```

**Review by csett86 — CHANGES_REQUESTED** [review]
Please see inline, just small consistency topics

**Review by csett86 — APPROVED** [review]

**Comment by csett86** [comment]
Thanks a lot for your contribution @Waboodoo

**Comment by csett86** [comment]
I‘ll just let the CI run through, if that all looks good, I‘ll squash and merge it

**Comment by Waboodoo** [comment]
You're welcome 👍🏻

**Pull request merged**